src/trayitem.c

```diff
@@ -42,6 +42,18 @@ void tray_item_update ( GtkWidget *self )
       gtk_widget_set_name(priv->icon,"tray_passive");
       sni_item_set_icon(priv->sni,SNI_PROP_ICON,SNI_PROP_ICONPIX);
     }
+    if(priv->sni->string[SNI_PROP_LABEL]!=NULL
+      && priv->sni->string[SNI_PROP_LABEL][0] != '\0')
+    {
+      gtk_label_set_markup(GTK_LABEL(priv->label),
+        priv->sni->string[SNI_PROP_LABEL]);
+      gtk_widget_show(priv->label);
+    }
+    else
+    {
+      gtk_label_set_text(GTK_LABEL(priv->label), "");
+      gtk_widget_hide(priv->label);
```

Comment:
The additional `gtk_label_set_text()` is a bad workaround because `gtk_widget_hide()` wasn't working and I didn't immediately find the reason of that (might be some `gtk_widget_show_all()` somewhere). Just another reason this PR is a Draft one.

Reply:
Sfwbar handles hiding/showing widgets via a CSS property, calling gtk_widget_show_hide gets undone the next time css_widget_cascade is called. The solution would be to either apply a CSS style change or to remove it from the container when not needed (in this case we will need to ref the label and unref it on parent widget destruction).

Reply:
Ah, that totally makes sense. Will change it to just set/remove the CSS class.

Reply:
Ended up adding a css class instead of setting the name via `gtk_widget_set_name()`.
That way the label can still have its own name if deemed necessary in the future.

Also added a default
```css
.hidden {
	-GtkWidget-visible: false;
}
```
and two small wrappers to `css.c`
- `css_add_class(widget, css)`
- `css_remove_class(widget, css)`

Reply:
This is really neat and can be used as a standard approach for hiding widgets going forward.